core/src/main/scala/shapeless/ops/tuples.scala

```diff
@@ -260,17 +260,17 @@ object tuple {
    *
    * @author Miles Sabin
    */
-  trait Replacer[T, U, V] extends DepFn2[T, U] with Serializable
```

Comment:
This becomes apparent here as it is defined as a functions that takes a `T` and a `U`,